src/components/ResultsCount.tsx

```diff
@@ -73,7 +76,16 @@ function useResultsCount() {
     pluralForm: `${resultsCount} Results`,
     count: resultsCount
   });
-  return resultsCountText;
+
+  if (resultsCount > (limit)){
+    const paginateStart = offset + 1;
+    const paginateEnd = min([(offset + limit), resultsCount]);
```

Comment:
why do we need lodash for this? Math.min exists

Reply:
replaced